Source/com/drew/imaging/png/PngMetadataReader.java

```diff
@@ -212,17 +212,20 @@ private static void processChunk(@NotNull Metadata metadata, @NotNull PngChunk c
         } else if (chunkType.equals(PngChunkType.tIME)) {
             SequentialByteArrayReader reader = new SequentialByteArrayReader(bytes);
             int year = reader.getUInt16();
-            int month = reader.getUInt8() - 1;
+            int month = reader.getUInt8();
             int day = reader.getUInt8();
             int hour = reader.getUInt8();
             int minute = reader.getUInt8();
             int second = reader.getUInt8();
-            Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-            calendar.setTimeInMillis(0);
-            //noinspection MagicConstant
-            calendar.set(year, month, day, hour, minute, second);
             PngDirectory directory = new PngDirectory(PngChunkType.tIME);
-            directory.setDate(PngDirectory.TAG_LAST_MODIFICATION_TIME, calendar.getTime());
+            if (DateUtil.isValidDate(year, month - 1, day) && DateUtil.isValidTime(hour, minute, second)) {
```

Comment:
As TAG_LAST_MODIFICATION_TIME doesn't contain the time zone information, I believe you should not assume a specific time zone at parsing time. You can specify a time zone by calling getDate(int tagType, TimeZone timeZone), or getDate(int tagType) if you assume GMT.